Library/Homebrew/test/macos_version_spec.rb

```diff
@@ -158,30 +156,4 @@
     expect(described_class.new("10.0").unsupported_release?).to be true
     expect(described_class.new("1000").prerelease?).to be true
   end
-
-  describe "#requires_nehalem_cpu?", :needs_macos do
-    context "when CPU is Intel" do
-      it "returns true if version requires a Nehalem CPU" do
-        allow(Hardware::CPU).to receive(:type).and_return(:intel)
-        expect(described_class.new("10.14").requires_nehalem_cpu?).to be true
-      end
-
-      it "returns false if version does not require a Nehalem CPU" do
-        allow(Hardware::CPU).to receive(:type).and_return(:intel)
-        expect(described_class.new("10.12").requires_nehalem_cpu?).to be false
-      end
-    end
-
-    context "when CPU is not Intel" do
-      it "raises an error" do
-        allow(Hardware::CPU).to receive(:type).and_return(:arm)
-        expect { described_class.new("10.14").requires_nehalem_cpu? }
-          .to raise_error(ArgumentError)
-      end
-    end
-
-    it "returns false when version is null" do
-      expect(described_class::NULL.requires_nehalem_cpu?).to be false
-    end
-  end
```

Comment:
Should we keep this given pre-Ventura versions do require Nehalem CPU?

Reply:
From my understanding, macOS Sierra was the first version of macOS that requires Nehalem at a minimum. Do we still need this if everything we support is Nehalem or later?

Reply:
Gotcha, yeah, this shouldn't be needed anymore then.